Dockerfile.assisted-service

```diff
@@ -33,10 +33,11 @@ RUN go mod download
 
 COPY . /assisted-service/
 
-RUN CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service cmd/main.go
-RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service-operator cmd/operator/main.go
-RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service-admission cmd/webadmission/main.go
-RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/agent-installer-client cmd/agentbasedinstaller/client/main.go
+RUN dnf install -y git && dnf clean all
```

Comment:
can we group this dependency install with [other installs](https://github.com/openshift/assisted-service/pull/4658/files#diff-c56f23b9a1fea65a75db47b3dbf186b3f3288928d23d964e799d8c09b362b87eL19)?